content/v2.1/en/introduction/what-is-kubesphere.md

```diff
@@ -1,51 +1,43 @@
 ---
 title: "What is KubeSphere"
-keywords: 'Kubernetes, docker, jenkins, devops, istio'
+keywords: 'Kubernetes, docker, jenkins, devops, istio, service mesh, devops, microservice'
```

Comment:
More keywords is not better, it is generally less than 5 keywords in most articles.